### ISSUE TITLE
Multiple electrolysers cause immense lag

### ISSUE THREAD

**Issue by Gombado** [turbodiesel4598/NuclearCraft#212]
Hey there, so I just hooked up a series ( 20 ) of electrolysers (to laminar fluidducts with enderium servos), each with 30 speed upgrades and the lag is **spectacular** , im assuming its graphical lag as the situation resolved when I fled to spawn.

Id love to give further information however after crashing within viewing distance of the electrolysers my minecraft now crashes when i try to log in.  Hope theres a fix for this!

**Comment by turbodiesel4598** [comment]
Have you got a crash report? How much power were you feeding the Electrolysers?

**Comment by Gombado** [comment]
Hey there turbo,

I looked up last nights logs and it just shows the launching messages and some chat - it seems the crashes didnt generate a report. It seemned to me that they were memory errors.

Today (because I have the fusion reactor chunkloaded) I could log in to a base without power. And without graphical lag. 

here is my setup: https://imgur.com/a/4ePof , the advanced powercells output 20k rf/t , and they were fully charged. I have the electrolysers equipped with 30 speed upgrades and unlimited water from the sink. 

Hope this helps, building the fusion reactor was great fun (I think it's an amazing mod, great work!)

**Comment by Gombado** [comment]
An update with my observations and a history of what happened, let me preface this with that I have no knowledge of minecraft or the nuclearcraft mod or its dependenancies:

The server had strange crashes yesterday evening due to a vulpes library error resulting in most players not being able to log in (with a fatal error). I had to reinstall the modpack (through twitch) and had  the 1 fps when I did manage to login. I decided to go to bed and let it run dry and stop overnight.

when I logged back in all the batteries were empty and I had no lag, I did however find a problem with my piping which meant hydrogen wasnt going anywhere,that combined with the limitless laminar ducts mustve meant there was a lot _lot_ **lot** of fluid in there. 

So I fixed the piping,fired the system up again with the added electrolysers (32 total now) 30 speed upgrades each which means they require just below 20k rf/t (which the advanced power cells supply per side) **_and to both my surprise and joy the electrolysers was now running smoothly_**

So my running theory is that I dun goofed and had a laminated fluid pipe filled to the brim with hydrogen and that caused crazy graphic lag. Or that it was some sort of server side issue caused by some vulpes library.

So my apologies for wasting your valuable freely given time and let me take this moment once more to thank you for making this fantastic mod, you rock! 

Here's some pictures I made of the reactor in action: https://imgur.com/a/RiHbT
